setup.py

```diff
@@ -20,9 +20,13 @@ def get_version() -> str:
 
 extras = {}
 
+extras["torch"] = [
+    "torch",
+]
+
 extras["testing"] = [
     "pytest",
-]
+] + extras["torch"]
```

Comment:
is there value in having a second CI setup with *no torch*? (to make sure we don't introduce a dependency on torch inadvertently down the road)

Reply:
Yes, I can add that